### ISSUE TITLE
even after disabling automatic updates, on Android a request to CloudFront is made.

### ISSUE THREAD

**Issue by KarthikSiddarth** [expo/expo#3403]
### Environment

Expo CLI 2.7.1 environment info:
    System:
      OS: Linux 3.10 CentOS Linux 7 (Core)
      Shell: 4.2.46 - /bin/bash
    Binaries:
      Node: 10.9.0 - /usr/local/node/bin/node
      npm: 6.7.0 - /usr/local/node/bin/npm
    npmPackages:
      expo: ^28.0.0 => 28.0.1 
      react: 16.3.1 => 16.3.1 
      react-native: https://github.com/expo/react-native/archive/sdk-28.0.0.tar.gz => 0.55.4 
      react-navigation: ^1.0.0-beta.21 => 1.6.1 
    npmGlobalPackages:
      expo-cli: 2.7.1

App Target: Android

### Steps to Reproduce

We built the entire app using Expo SDK, we found that app startup time to too high, goes till 20 seconds, so we configured automatic updates in app.json. The configurations are mentioned below:
{
  "updates": {
    "enabled": false,
    "checkAutomatically": ‘ON_LOAD’,
    "fallbackToCacheTimeout": 0
  }
}

1. Configure Proxy manually
2. Open the app
3. Intercept the requests made by the app using any HTTP interceptor (We used Charles for Mac)
4. You can find a request to CloudFront is made.

### Expected Behavior

No Matter what connection is the phone be on (WiFi, 3G, 4G, 2G), if the automatic updates are disabled, the app should open in 3 to 4 seconds.

(Write what you thought would happen.)

### Actual Behavior

After configuration, the updates were deployed through the hot deployment channel, changed the connection type to one of these mentioned types (WiFi, 4G, 3G, 2G). If the connection is set to WiFi, 4G, 3G, the app opens in 3 to 4 seconds but if the connection is set to 2G, the app takes around 20 seconds to open. We configured a proxy manually and intercepted the request using a Web Debugging Proxy (Charles), we found that a request to CloudFront CDN is being made.
If the connection type is set to WiFi, 3G, 4G, the data from CloudFront gets downloaded faster (around 150 KB) but when the connection type is set to 2G the download of the data is slower. The problem is the app opens only after the data is downloaded.
We intercepted the request from iPhone and found no request to CloudFront is made.

Link to the app: https://play.google.com/store/apps/details?id=com.stanzaliving.app

**Comment by sjchmiela** [comment]
Hey, @KarthikSiddarth!

Could you please provide some more info on the request that is being made to CloudFront? I just ejected an SDK32 application to reproduce the issue and the only request to CloudFront that has been issued was https://d1wp6m56sqw74a.cloudfront.net/~assets/fa6577fecc0a7838f15a254577639984, i.e. a request for an image that will be used as the icon for your app in "Recent apps" view triggered by:
- https://github.com/expo/expo/blob/0fc89eaf350f1a97f890cba3339103ff086a99e3/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java#L490
- https://github.com/expo/expo/blob/0fc89eaf350f1a97f890cba3339103ff086a99e3/android/expoview/src/main/java/host/exp/exponent/utils/ExperienceActivityUtils.java#L90
- which calls an `AsyncTask` in `ExponentManifest.loadIconBitmap` https://github.com/expo/expo/blob/0fc89eaf350f1a97f890cba3339103ff086a99e3/android/expoview/src/main/java/host/exp/exponent/ExponentManifest.java#L622

This shouldn't be blocking the app. 🤔 Maybe this issue got solved in newer versions of Expo SDK?

**Comment by manoj-makkuboy** [comment]
Hey @sjchmiela I am having the similar issue in sdk 31. After your comment I tried updating my application to 32 and my app started to break. So sticking with sdk 31 for now. Can you try the same thing in 31

**Comment by AdamJNavarro** [comment]
Hey all,

In an effort to create a clearer picture for our team about what is still actionable, we are going through older and inactive issues that were missing the necessary information for us to attempt to debug (see: Needs more info label).

If anyone is still encountering this issue on a presently supported SDK version (30-34) and has Minimal, Complete and Verifiable Example (MCVE) and better yet, the root cause identified, we encourage you to create a new issue with said information!

Cheers,
Adam
